package.json

```diff
@@ -22,8 +22,9 @@
     "style:fix": "yarn style --write",
     "test": "jest --coverage --group=unit",
     "test:ci": "yarn test --ci",
-    "test:integration:run": "jest --group=integration -b -w 3 --silent",
-    "test:integration": "{ lsof -i :9944 > /dev/null && yarn test:integration:run; } || echo 'Can not connect to chain. Is it running?'; exit 1;",
+    "test:integration": "jest --group=integration -w 3 --testTimeout=30000",
```

Comment:
Q: why do we need to add a timeout? Is the default not sufficient?

Reply:
also the `-b` option was removed. Why is that?

Reply:
Oh, yes, I figured we don't actually want it to bail (fail fast) unless we test in ci mode, because that means we wouldn't known what else is broken. At least it was annoying me quite a bit.

Reply:
On the timeout, it does work without this but I though 30s is a reasonable default for our integration tests. And because we don't have a separate jest config for integration testing, I set it here.